### ISSUE TITLE
 Additional Custom Breakpoints and sticky elements

### ISSUE THREAD

**Issue by Goodimp** [elementor/elementor#16134]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Description

With the additional breakpoint option activated (I only added tablet extra) if you try to apply the motion effects sticky feature to an element only the original 3 breakpoints are available as options in the Sticky On field.

## Steps to reproduce

See description
## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.35
	PHP Version: 7.4.22
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8
	Site URL: https://gicms.co.uk
	Home URL: https://gicms.co.uk
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: 1
	Debug Mode: Inactive

== Theme ==
	Name: GI Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.6.8
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36

== Active Plugins ==
	Astra Pro
		Version: 3.5.8
		Author: Brainstorm Force

	Elementor
		Version: 3.4.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.3.8
		Author: Elementor.com

	ManageWP - Worker
		Version: 4.9.9
		Author: GoDaddy

	Password Protected
		Version: 2.4
		Author: Ben Huson

	Superfly Menu
		Version: 5.0.19
		Author: Looks Awesome

	The Grid
		Version: 2.7.9.1
		Author: Themeone

	Ultimate Addons for Elementor
		Version: 1.33.2
		Author: Brainstorm Force

	Wordfence Security
		Version: 7.5.5
		Author: Wordfence

	Yoast Duplicate Post
		Version: 4.1.2
		Author: Enrico Battocchi & Team Yoast


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 
		Author: GoDaddy


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Ultimate Addons for Elementor: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Ultimate Addons for Elementor: Compatible

== Elements Usage ==
	
	footer : 1
		heading : 1
		image : 1
		text-editor : 8
	header : 1
		image : 1
	wp-page : 6
		bdt-post-gallery : 2
		bdt-post-grid : 2
		button : 20
		divider : 2
		flip-box : 4
		heading : 39
		icon : 1
		icon-list : 14
		image : 9
		image-box : 8
		image-carousel : 1
		spacer : 10
		testimonial-carousel : 1
		text-editor : 29
		uael-image-gallery : 4


== Log ==
	
Log: showing 20 of 402021-08-09 15:23:37 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2021-08-09 15:23:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.3.1',
)]
2021-08-09 15:24:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.1',
  'to' => '3.3.5',
)]
2021-08-18 12:25:16 [info] elementor::elementor_updater Started 
2021-08-18 12:25:16 [info] Elementor/Upgrades - _on_each_version Start  
2021-08-18 12:25:17 [info] Elementor/Upgrades - _on_each_version Finished 
2021-08-18 12:25:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.1',
  'to' => '3.4.0',
)]
2021-08-18 12:25:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.1',
  'to' => '3.4.0',
)]
2021-08-18 16:02:04 [info] elementor::elementor_updater Started 
2021-08-18 16:02:04 [info] Elementor/Upgrades - _on_each_version Start  
2021-08-18 16:02:04 [info] Elementor/Upgrades - _on_each_version Finished 
2021-08-18 16:02:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.0',
  'to' => '3.4.1',
)]
2021-08-18 16:02:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.0',
  'to' => '3.4.1',
)]
2021-08-20 13:37:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.5',
  'to' => '3.3.7',
)]
2021-08-20 13:37:30 [info] elementor::elementor_updater Started 
2021-08-20 13:37:30 [info] Elementor/Upgrades - _on_each_version Start  
2021-08-20 13:37:30 [info] Elementor/Upgrades - _on_each_version Finished 
2021-08-20 13:37:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.1',
  'to' => '3.4.2',
)]
2021-08-20 13:37:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.1',
  'to' => '3.4.2',
)]
2021-08-24 14:28:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.7',
  'to' => '3.3.8',
)]

PHP: showing 3 of 3PHP: 2021-05-04 15:10:03 [notice X 5][/home/gicmsco/public_html/wp-content/plugins/elementor/includes/managers/image.php::86] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-05-16 14:09:25 [notice X 3][/home/gicmsco/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-07-21 09:39:50 [notice X 1][/home/gicmsco/public_html/wp-content/plugins/elementor-pro/license/updater.php::75] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
</details>

**Comment by maurocarreraok** [comment]
+1

**Comment by DorShahar** [comment]
Hi @Goodimp 
Thank you for taking the time and reporting this.
We fixed the issue in Pro 3.4.1 - please make sure you updated to the recent version: 
![image](https://user-images.githubusercontent.com/19154938/134356880-e737409e-d6a2-4831-9445-25003aae54e7.png)


Thanks.
